src/analyses/libraryFunctions.ml

```diff
@@ -167,23 +167,34 @@ let invalidate_actions = [
     "GetSpinlock", readsAll;
     "ReleaseSpinlock", readsAll;
     "atoi", readsAll;             (*safe*)
+    "__builtin_ctz", readsAll;
+    "__builtin_ctzl", readsAll;
+    "__builtin_ctzll", readsAll;
+    "__builtin_clz", readsAll;
     "bzero", writes [1]; (*keep 1*)
     "connect", readsAll;          (*safe*)
     "fclose", readsAll;           (*safe*)
     "fflush", writesAll;          (*unsafe*)
     "fopen", readsAll;            (*safe*)
+    "fdopen", writes [1];            (*safe*)
```

Comment:
According to the man page, the signature is `FILE *fdopen(int fd, const char *mode)`, so it should only read its arguments.